### ISSUE TITLE
Deconvolution / loss of MSMS Pattern

### ISSUE THREAD

**Issue by ElMouth** [mzmine/mzmine2#280]
Hello,

I noticed that when I proceed the deconvolution step (with Wavelets or Local Minimum Search), some of the features enlisted (that possessed an MS/MS in the non-deconvoluted chromatograms peak list) loose their MS/MS pattern... Even high intensity ions...
Does someone have an idea what is happening and how to avoid that?

Many thanks!

**Comment by tomas-pluskal** [comment]
Hi,
Check the retention time of the MS/MS scan. Is it within the retention time span of the deconvoluted peak?

**Comment by ElMouth** [comment]
Hi Tomas,

Yes MSMS scans are still within RT ranges of my deconvoluted peaks. However, by varying parameters during deconvolution and chromatogram building steps, I managed to get better results (not optimal but satisfying).
I also noticed that I can have this issue when I perform deconvolution on smoothed data.

How does the algorithm select the MS2 scan to be associated to each feature? Is it the most intense MS2 spectra in the RT range?
I don't know if it's (easily) feasible but it would be great if associated MSMS patterns could be averaged over the retention time span of the deconvoluted features detected! It might reduce background noise, alterations in signal intensity or drift in mass accuracy. A word to the wise ;)

Thanks very much for your help!

**Comment by tomas-pluskal** [comment]
The MS2 is selected as the most intense spectrum within the RT range and with the precursor mass matching the peak m/z. If you send me an example file and the exact parameters you used for processing, I can investigate why your MS2 spectrum was not linked.

**Comment by RicardoMBorges** [comment]
Hello, I've just came across the same issue.
Eventually, when I marked the 'm/z range for MS2 scan pairing (Da)' and the 'RT range for MS2 scan pairing (min)', I got the MS2 patterns back.

Can I get this opportunity to ask exactly what is the parameter wavelets from the ADAP algorithm? ...I read a paper on wavelets form chromatogram, but no success in understanding it.

Thanks
Ricardo M Borges
